### PR TITLE
chore: add installation and setup docs.

### PR DESCRIPTION
**Pull request by i4ki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+Finally, make sure that the `terramate-ls` binary is available on your PATH.
+This process will differ depending on your operating system.
+
+### Setup in the code editor
+
+At the moment only [vscode](https://code.visualstudio.com/) is officially
+supported by the [vscode-terramate](https://github.com/mineiros-io/vscode-terramate)
+extension.
+
+The only `terramate-ls` specific setup required is making sure it installed in a
```
Comment (katcipis) [inline README.md]:
```suggestion
The only `terramate-ls` specific setup required is making sure it is installed in a
```

**Review by katcipis — COMMENTED** [review]
LGTM + small possible typo

**Review by katcipis — DISMISSED** [review]

**Comment by i4ki** [comment]
waiting for the project's name change to merge this.

**Comment by i4ki** [comment]
The docs mention installing using release binaries but there's no release yet.
I'm going to make the very first release after merging this.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
-Terramate Language Server.
+The Terramate Language Server provides features to any [LSP](https://microsoft.github.io/language-server-protocol/)-compatible code editor.
+
+## Getting Started
+
+### Installing
+
+#### Using Go
+
+To install using Go just run:
```
Comment (soerenmartius) [inline README.md:11]:
```suggestion
To install using Go run:
```

**Review by soerenmartius — APPROVED** [review]
lgtm, minor suggestion

**Review by katcipis — APPROVED** [review]

**Pull request merged**